### PR TITLE
Monitored tokens API group

### LINKED ISSUE #12: Setup API Group: Monitored Tokens
Reference

- [x] GET | Get list of monitored tokens | https://api.circle.com/v1/w3s/config/entity/monitoredTokens
- [x] GET | Retrieve monitored tokens | https://api.circle.com/v1/w3s/config/entity/monitoredTokens
- [x] POST | Set monitored tokens |  
- [x] PUT | Update monitored tokens |  
- [x] PUT | Update monitored tokens scope | https://api.circle.com/v1/w3s/config/entity/monitoredTokens/scope
- [x] POST | Delete monitored tokens | https://api.circle.com/v1/w3s/config/entity/monitoredTokens/delete

### PR DESCRIPTION
**Pull request by avkos**
Fixes #12

### CODE REVIEW

**Review by danforbes — CHANGES_REQUESTED** [review]
Can you give me some time to update the documentation before this is merged?

**Comment by avkos** [comment]
> Can you give me some time to update the documentation before this is merged?

yes. for sure!

File: packages/web3-circle-sdk/src/types.ts [hunk 1]
```diff
@@ -47,13 +56,31 @@ export type FilterOptions = {
...
 
+export type GetMonitoredTokensParameters = {
+  blockchain?: BLOCKCHAIN;
+  tokenAddress?: string;
+  symbol?: string;
+  from?: string; // ISO 8601 date-time format
+  to?: string; // ISO 8601 date-time format
+  pageBefore?: string; // UUID
+  pageAfter?: string; // UUID
+  pageSize?: number; // between 1 and 50, default is 10
```
Comment (danforbes) [inline packages/web3-circle-sdk/src/types.ts]: Why isn't this type just composing `FilterPagesOptions` like the other types?

File: packages/web3-circle-sdk/src/types.ts [hunk 2]
```diff
@@ -1703,3 +1730,8 @@ export type RegisteredEntity = {
   /** System-generated unique identifier of the entity's app. */
   recoveryFile: string;
 };
+
+export type MonitoredTokenEntity = {
+  tokens: Token[];
+  scope: string;
```
Comment (danforbes) [inline packages/web3-circle-sdk/src/types.ts:1810]: Should this be the `MONITORED_TOKENS_SCOPE` enum instead of a string?

**Review by danforbes — APPROVED** [review]
Just a couple of questions. Please review [my recent commit](https://github.com/ChainSafe/web3-circle-libs/pull/53/commits/98cd387f5b16b6fbcac6a854d51be01c6cb88250) as I did make one small change to `index.ts` (changed `CircleSdk.monitoredTokensApi` to `CircleSdk.monitoredTokens` for consistency).

**Pull request merged**